Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -778,6 +812,31 @@ static std::string getCacheDatabaseName() {
   return TCHAR_TO_UTF8(*PlatformAbsolutePath);
 }
 
+void ACesium3DTileset::UpdateFromView(FSceneViewFamily& ViewFamily) {
+  // TODO: save reference? How expensive is this?
+  // This is called separately for each view, may need to optimize this
+  UCesiumBoundingVolumePoolComponent* pBoundingVolumePool =
+      this->FindComponentByClass<UCesiumBoundingVolumePoolComponent>();
```

Comment:
This method loops over the actor's `OwnedComponents` and calls `IsA` on each. If we have 500 loaded tiles and two cameras, we're iterating through 1000 components per frame and calling IsA on them. I haven't profiled to prove that's a bad idea, but it sounds pretty inefficient. And I think it should be easy and much more efficient to just store the `UCesiumBoundingVolumePoolComponent` in a transient UProperty.

Reply:
Applied this suggestion!